README.md

```diff
@@ -1,6 +1,6 @@
 ## Installation
```

Comment:
Doesn't seem like we need any of these changes

Reply:
God damn auto save format.